frontend/game.css

```
@@ -4,20 +4,31 @@
   margin: 0 auto;
 }
 
+#infoContent {
+  font-family: system, -apple-system, BlinkMacSystemFont, 'Helvetica Neue',
+    'Lucida Grande';
+  border-bottom: 1px #eee solid;
+  display: grid;
+  grid-auto-flow: column;
+  grid-gap: 2em;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 1em;
+}
+
 #share {
   color: #888;
-  margin-bottom: 2em;
   font-size: 0.8em;
-  padding-bottom: 2em;
-  border-bottom: 1px #eee solid;
-  font-family: system, -apple-system, BlinkMacSystemFont, 'Helvetica Neue',
-    'Lucida Grande';
 }
 
 #share .url {
   color: #888;
 }
 
+#timer {
+  font-size: 3em;
+}
```

```suggestion
  font-size: 1.5em;
  font-family: "Courier New", monospace;
}
```

I think we'll wantthe timer to be a monospace font so that its width doesn't flicker as it counts down. Courier New will match the top header. Also, IMO, the size was a bit too large.